mrp_sale_info/__openerp__.py

```diff
@@ -1,10 +1,11 @@
 # -*- coding: utf-8 -*-
 # © 2016 Antiun Ingenieria S.L. - Javier Iniesta
+# © 2016 Elico Corp (https://www.elico-corp.com).
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 {
     "name": "MRP Sale Info",
     "summary": "Adds sale information to Manufacturing models",
-    "version": "9.0.1.0.0",
+    "version": "9.0.1.1.0",
     "category": "Manufacturing",
     "website": "http://www.antiun.com",
     "author": "Antiun Ingeniería S.L., "
```

Comment:
OK, this is because an absurd limitation of 128 characters on this field. I opened an issue about it some time ago: https://github.com/odoo/odoo/issues/5850. Please support me in it.